meson.build

```diff
@@ -298,6 +298,10 @@ if checks_level == 9999
   endif
 endif
 
+if get_option('buildtype').startswith('debug')
+  add_project_arguments('-DRZ_BUILD_DEBUG', language : 'c')
+endif
```

Comment:
Let's put this thing in rz_userconf.h.in and set it similar to the other variables there. That way people using the API can also see if rizin was compiled in debug mode or not.